### DISCUSSION TITLE
How does one can list all spelling errors in a project in NetBeans?

### DISCUSSION THREAD

**Discussion by jonathanpoulin9** [apache/netbeans#4247] [Q&A]
NetBeans has a spellchecker. I am using the latest version, 14. I would like to know if this is possible to scan all files (in my case, .java files) in a project and list the spelling errors (in Java comments) in one place. It can be a plugin as well. Otherwise, I need to open each files to look for the red underlines.

**Comment by mbien** [comment]
I don't think that this is currently possible. Would be a nice feature though if someone wants to implement it.

**Comment by jonathanpoulin9** [comment]
@mbien Yes, I think too. That would be a real nice feature to add. In that case, am I better to add a new issue?

**Reply by mbien** [reply]
@jonathanpoulin9 you can add a feature request if you want. But there is no lack of issues so I don't know if anyone will want to implement that. But I can always tag it with "contribution welcome".

**Reply by jonathanpoulin9** [reply]
Ok perfect. So I just added a new issue as a feature request. Here it is: #4253. And yes please tag it as "contribution welcome". Thanks! :)
